### ISSUE TITLE
sanitize_fname() behaves incorrectly on Windows

### ISSUE THREAD

**Issue by csunday95** [sirMackk/py3tftp#2]
On windows, due to the explicit use of the forward slash in file_io.py line 13, the path for a given requested file is resolved incorrectly. For example, supplying an unqualified filename results in `b'C:\\filename'` regardless of the current working directory. A possible cross-platform replacement would be 

```python
import os
import os.path as opath


def sanitize_fname(fname):
    """
    Ensures that fname is a path under the current working directory.
    """
    root_dir = os.getcwd().encode('ascii')
    return opath.join(root_dir, opath.normpath(fname))
```

**Comment by sirMackk** [comment]
Hi @csunday95, thanks for spotting this and posting some code! I think I'll be able to get around to including this fix and making a new release sometime around in January or February. You are of course more than welcome to create a PR if you'd want to see this fix included sooner :).

**Comment by sirMackk** [comment]
Closed by #15
